### DISCUSSION TITLE
usb boot issue

### DISCUSSION THREAD

**Discussion by Kirjoittamaton** [Nitrux/nitrux-bug-tracker#157] [Q&A]
usb live boot just freezes after going to boot, info when i selected nomodeset boot option, some issue with atomisp-isp2, pc is lenovo 320miix-10cr 2in1.

![IMG_20240111_152250](https://github.com/Nitrux/nitrux-bug-tracker/assets/128027949/690a35a1-7659-4303-ba20-dd6ced8661f3)

**Comment by UriHerrera** [comment]
Please describe the problem, including what you've tried and how the problem is reproducible, and include adequate information about the hardware so this can be moved as an issue.

**Reply by Kirjoittamaton** [reply]
tried remake boot media, tried to boot with other pc and that booted, cpu: intel atom Intel Atom x5-Z8350, 4gb of ram, 128gb emmc, intel cherry trail hd graphics.  it has detacable touchscreen.

**Reply by UriHerrera** [reply]
The file mentioned in your screenshot is indeed missing. The firmware file is for the webcam.

You mentioned that the "USB live boot just freezes after going to boot." What do you mean by that? Do you not get to the desktop?

**Reply by Kirjoittamaton** [reply]
it just stays there, not going any further

**Reply by UriHerrera** [reply]
Please provide thorough information about the problem, i.e., how you are creating the Live USB and the precise hardware configuration of your laptop, as I can't reproduce the problem given I don't have your laptop. The firmware file mentioned in the screenshot was added in the latest release. Nonetheless, that firmware file should not've been enough not to display the desktop.

Also, you mentioned your picture was taken after using `nomodeset`. This parameter instructs the Linux kernel to turn off the video mode setting during the boot process. That means your computer will not load a graphical session when you boot.

Boot using the regular option.
